pkg/virt-launcher/virtwrap/manager.go

```diff
@@ -1365,6 +1367,16 @@ func (l *LibvirtDomainManager) GetGuestInfo() (v1.VirtualMachineInstanceGuestAge
 	return guestInfo, nil
 }
 
+// GetInterfaceStatus returns the interfaces Guest Agent reported
+func (l *LibvirtDomainManager) GetInterfaceStatus() []api.InterfaceStatus {
```

Comment:
There is no need to the `Get` prefix, a simple `InterfacesStatus` is enough. (BTW, it's plural)